test/benchmarks/driver_bench/src/main.mts

```diff
@@ -198,6 +199,33 @@ function calculateCompositeBenchmarks(results: MetricInfo[]) {
   return [...results, ...compositeResults];
 }
 
-const finalResults = calculateCompositeBenchmarks(results);
+function calculateNormalizedResults(results: MetricInfo[]): MetricInfo[] {
+  const primesBench = results.find(r => r.info.test_name === 'primes');
+  const pingBench = results.find(r => r.info.test_name === 'ping');
+
+  if (pingBench) {
```

Comment:
We may want to take some precaution on this being optional, currently it should always be here, you can make a tight type narrowing with: `assert.ok(pingBench)` 

In the future we want to be able to filter to some test for quick local checking so we'll need to revisit this then, but for now I think we want to know if this is missing.

Reply:
If we're alerting on normalized metrics maybe it makes sense to never _not_ run `primes` and `ping`